src/CMake/embedded_system.cmake

```diff
@@ -108,7 +108,7 @@ message("-- Preparing XRT pkg-config")
 set(XRT_PKG_CONFIG_DIR "/usr/lib/pkgconfig")
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`